parser/src/main/java/log/parser/service/impl/MatchServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package log.parser.service.impl;
+
+import java.security.SecureRandom;
+import java.util.ArrayList;
+
+import log.parser.service.MatchService;
+
+public class MatchServiceImpl implements MatchService {
+
+//	@Autowired
+//	private MatchRepositoryImpl repo;
+	
+	
+	@Override
+	public Long ingestMatch(String payload) {
+		SecureRandom random = new SecureRandom();
+		long matchId = Math.abs(random.nextLong());
```

Comment:
matchId generation should use a UUID to prevent collissions

Reply:
I used UUID generator and changed matchId variable type to String.